src/coprocessor/dag/rpn_expr/types/expr_builder.rs

```diff
@@ -56,6 +54,50 @@ impl RpnExpressionBuilder {
         )?;
         Ok(RpnExpression::from(expr_nodes))
     }
+
+    /// Creates a new builder instance.
+    ///
+    /// Only used in tests. Normal logic should use `build_from_expr_tree`.
+    #[cfg(test)]
```

Comment:
So many test related functions. Can we move these functions into somewhere else?

Reply:
Or we can remove "cfg(test)". These functions are perfectly RpnExpressionBuilder functions. They have a `cfg(test)` only because normal logic doesn't use them currently.

Reply:
prefer to move them into another place

Reply:
Please suggest a new name better than "RpnExpressionBuilder"